Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
What file is `python --version` getting the number from originally? I'd use that, since it'll be available whenever as long as the source has been unpacked, which is why I asked what "pre-build" means.